schema/RSRESISTANCEWATERHEATSOURCE.schema.yaml

```diff
@@ -51,3 +51,10 @@ Performance:
       Data Type: "Numeric"
       Units: "W"
       Required: True
+    resistance_lockout_temperature_hysteresis: # Was hysteresis_temperature_difference in HeatSourceConfiguration
+      Description: "Hysteresis for resistance lockout"
+      Data Type: Numeric
+      Units: "K"
+      Required: False
+      Notes:
+        - "Resistance hysteresis for lockout"
```

Comment:
Where is this used? Resistance heaters don't "lock out" the same way condensers do, but I can imagine it's used to approximate some kind of behavior?

Reply:
Ask Ben about this.

Reply:
I had reached the mistaken conclusion that this was needed, but after further testing, it seems that it does not affect any regressions. But it must have seemed necessary at some point, because values are entered for most models with resistance heat sources.